src/utils/storage/index.ts

```diff
@@ -94,6 +95,7 @@ const StorageUtils = {
   lastSelectedMenu: storageBuilder(StorageKeysEnum.LastSelectedMenu),
   blockingStatus: storageBuilder(StorageKeysEnum.BlockingStatus),
   blockedStatus: storageBuilder(StorageKeysEnum.BlockedStatus),
+  incognitoCreatePostFirstTime: storageBuilder(StorageKeysEnum.IncognitoCreatePostFirstTime),
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new storage key `IncognitoCreatePostFirstTime` is correctly built using the `storageBuilder` function and added to the `IStorageUtils` interface. This allows it to be used consistently with other storage keys throughout the application.
```diff
+   incognitoCreatePostFirstTime: storageBuilder(StorageKeysEnum.IncognitoCreatePostFirstTime),
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->